data/35e/wizards_of_the_coast/supplement/deities_and_demigods/dd_templates.lst

```diff
@@ -2,7 +2,7 @@
 
 ###Block: Creature Subtypes
 # Template Name	Unique Key				Hit Dice Size	Visible	Damage Reduction	Template																										Source Page			Define Stat			Multiple Requirements														Required Race		Combat bonus				Misc bonus		Stat bonus							Modify VAR							Ability																																Spell Res.
-Mummy (Greater)						HITDIE:12		VISIBLE:YES				TEMPLATE:CHOOSE:Slam (Fine)|Slam (Diminutive)|Slam (Tiny)|Slam (Small)|Slam (Medium)|Slam (Large)|Slam (Huge)|Slam (Gargantuan)|Slam (Colossal)	TEMPLATE:Undead	SOURCEPAGE:DD		DEFINESTAT:NONSTAT|CON																						BONUS:COMBAT|AC|8|TYPE=NaturalArmor	BONUS:MISC|CR|2	BONUS:STAT|STR|4	BONUS:STAT|INT,WIS,CHA|2									ABILITY:Internal|AUTOMATIC|Race Traits ~ Greater Mummy|Race Traits ~ Mummy
+Mummy (Greater)						HITDIE:12		VISIBLE:YES				TEMPLATE:CHOOSE:Slam (Fine)|Slam (Diminutive)|Slam (Tiny)|Slam (Small)|Slam (Medium)|Slam (Large)|Slam (Huge)|Slam (Gargantuan)|Slam (Colossal)	TEMPLATE:Undead	SOURCEPAGE:DD		DEFINESTAT:NONSTAT|CON																						BONUS:COMBAT|AC|8|TYPE=NaturalArmor	BONUS:MISC|CR|2	BONUS:STAT|STR|4	BONUS:STAT|INT,WIS,CHA|2									ABILITY:Internal|AUTOMATIC|Race Traits ~ Greater Mummy|Race Traits ~ Greater Mummy
```

Comment:
I still do not understand the purpose of this change:
ABILITY:Internal|AUTOMATIC|Race Traits ~ Greater Mummy|Race Traits ~ Greater Mummy which translates to:

ABILITY:Internal|AUTOMATIC|Race Traits ~ Greater Mummy
ABILITY:Internal|AUTOMATIC|Race Traits ~ Greater Mummy